### DISCUSSION TITLE
Should this be llm.cpp not llama.cpp?

### DISCUSSION THREAD

**Discussion by lin-calvin** [ggml-org/llama.cpp#3844] [Ideas]
though it is aim at running llama at the begin but it is able to run various non-llama models. so should it be renamed to llm.cpp to show that it can not only run llama？

**Comment by Green-Sky** [comment]
I think this was discussed long ago, and the conclusion pretty much was, that it's too late to rebrand. :)

**Reply by Green-Sky** [reply]
https://github.com/ggerganov/llama.cpp/discussions/2102#discussioncomment-6357101
